Classes/ARLocalTiledImageDataSource.m

```diff
@@ -11,16 +11,51 @@
 
 @implementation ARLocalTiledImageDataSource
 
+- (id)init
+{
+    self = [super init];
+    if (self) {
+        _extralVisualLevel = 1;
+    }
+    return self;
+}
+
 - (UIImage *)tiledImageView:(ARTiledImageView *)imageView imageTileForLevel:(NSInteger)level x:(NSInteger)x y:(NSInteger)y
 {
-    NSString *filename = [NSString stringWithFormat:@"%@/%@/%@_%@.%@", self.tileBasePath, @(level), @(x), @(y), self.tileFormat];
-    return [UIImage imageWithContentsOfFile:filename];
+#define _PATH(_LEV, X, Y)   [NSString stringWithFormat:@"%@/%@/%@_%@.%@", self.tileBasePath, @(_LEV), @(X), @(Y), self.tileFormat];
```

Comment:
Just make a separate function for this if it's to be reused, defines are not local to a function if I am not mistaken, so this is actually global.